### ISSUE TITLE
bug: OT2 camera not responding

### ISSUE THREAD

**Issue by ChrisTwig** [Opentrons/opentrons#14814]
### Overview

I tried the /camera/picture endpoint and recieved.
{
  "message": "Error 1000 COMMUNICATION_ERROR (CameraException): Failed to communicate with camera",
  "errorCode": "4000"
}

running `ffmpeg -y -f video4linux2 -s 640x480 -i /dev/video0 -ss 0:0:1 -frames 1 /data/user_storage/image.jpg` gives ```[video4linux2,v4l2 @ 0x12af290] Cannot open video device /dev/video0: No such file or directory
/dev/video0: No such file or directory
```
errors

### Steps to reproduce

_No response_

### Current behavior

_No response_

### Expected behavior

_No response_

### Operating system

None

### System and robot setup or anything else?

_No response_

**Comment by koji** [comment]
@ChrisTwig 
sorry for the inconvenience to you. Software team is working on this issue.
The fix version will be released as 7.2.2.

**Comment by koji** [comment]
7.2.2 has been released. please try that.
https://github.com/Opentrons/opentrons/releases/tag/v7.2.2
